core/src/test/scala/kafka/tools/ReplicaVerificationToolTest.scala

```diff
@@ -55,7 +55,7 @@ class ReplicaVerificationToolTest {
 
     // If you change this assertion, you should verify that the replica_verification_test.py system test still passes
     assertTrue(s"Max lag information should be in output: `$output`",
-      output.endsWith(": max lag is 10 for partition a-0 at offset 10 among 3 partitions"))
+      output.endsWith(": max lag is 10 for partition a-1 at offset 10 among 3 partitions"))
```

Comment:
With TopicPartition changes, expectedReplicasPerTopicAndPartition  Map iteration order changed. Hence this change. System test is passing.